schemas/com.snowplowanalytics.snowplow/event/jsonschema/1-0-0

```diff
@@ -0,0 +1,488 @@
+		},
+		"br_features_quicktime": {
+			"type": ["boolean", "null"]
+		},
+		"br_features_realplayer": {
+			"type": ["boolean", "null"]
+		},
+		"br_features_windowsmedia": {
+			"type": ["boolean", "null"]
+		},
+		"br_features_gears": {
+			"type": ["boolean", "null"]
+		},
+		"br_features_silverlight": {
+			"type": ["boolean", "null"]
+		},
+		"br_cookies": {
+			"type": ["boolean", "null"]
+		},
+		"br_colordepth": {
+			"type": ["string", "null"],
+			"maxLength": 12
+		},
+		"br_viewwidth": {
+			"type": ["integer", "null"]
+		},
+		"br_viewheight": {
+			"type": ["integer", "null"]
+		},
+		"os_name": {
+			"type": ["string", "null"],
+			"maxLength": 50
+		},
+		"os_family": {
+			"type": ["string", "null"],
+			"maxLength": 50
+		},
+		"os_manufacturer": {
+			"type": ["string", "null"],
+			"maxLength": 50
+		},
+		"os_timezone": {
+			"type": ["string", "null"],
+			"maxLength": 255
+		},
+		"dvce_type": {
+			"type": ["string", "null"],
+			"maxLength": 50
+		},
+		"dvce_ismobile": {
+			"type": ["boolean", "null"]
+		},
+		"dvce_screenwidth": {
+			"type": ["integer", "null"]
+		},
+		"dvce_screenheight": {
+			"type": ["integer", "null"]
+		},
+		"doc_charset": {
+			"type": ["string", "null"],
+			"maxLength": 128
+		},
+		"doc_width": {
+			"type": ["integer", "null"]
+		},
+		"doc_height": {
+			"type": ["integer", "null"]
+		},
+		"tr_currency": {
+			"type": ["string", "null"],
+			"maxLength": 3
+		},
+		"tr_total_base": {
+			"type": ["number", "null"]
+		},
+		"tr_tax_base": {
+			"type": ["number", "null"]
+		},
+		"tr_shipping_base": {
+			"type": ["number", "null"]
+		},
+		"ti_currency": {
+			"type": ["string", "null"],
+			"maxLength": 3
+		},
+		"ti_price_base": {
+			"type": ["number", "null"]
+		},
+		"base_currency": {
+			"type": ["string", "null"],
+			"maxLength": 3
+		},
+		"geo_timezone": {
+			"type": ["string", "null"],
+			"maxLength": 64
+		},
+		"mkt_clickid": {
+			"type": ["string", "null"],
+			"maxLength": 128
+		},
+		"mkt_network": {
+			"type": ["string", "null"],
+			"maxLength": 64
+		},
+		"etl_tags": {
+			"type": ["string", "null"],
+			"maxLength": 500
+		},
+		"dvce_sent_tstamp": {
+			"type": ["string", "null"]
+		},
+		"refr_domain_userid": {
+			"type": ["string", "null"],
+			"maxLength": 128
+		},
+		"refr_dvce_tstamp": {
+			"type": ["string", "null"]
+		},
+		"domain_sessionid": {
+			"type": ["string", "null"],
+			"maxLength": 128
+		},
+		"derived_tstamp": {
+			"type": ["string", "null"]
+		},
+		"event_vendor": {
+			"type": ["string", "null"],
+			"maxLength": 1000
+		},
+		"event_name": {
+			"type": ["string", "null"],
+			"maxLength": 1000
+		},
+		"event_format": {
+			"type": ["string", "null"],
+			"maxLength": 128
+		},
+		"event_version": {
+			"type": ["string", "null"],
+			"maxLength": 128
+		},
+		"event_fingerprint": {
+			"type": ["string", "null"],
+			"maxLength": 128
+		},
+		"true_tstamp": {
+			"type": ["string", "null"]
+		}
+	},
+	"patternProperties": {
```

Comment:
I was thinking that we shouldn't have these self-describing properties at all and concentrate only on atomic properties (as we're doing this not for Analytics SDK, but for atomic tables), but right now actually I think maybe its not that bad idea. What do you think, @BenFradet?

Reply:
Maybe just make it `additionalProperties: true`?

Reply:
So far we don't want to validate anything using this schema, so I'm relatively relaxed about potential mistake.

Reply:
I think maybe it makes sense to rename it to `atomic` and drop self-describing entities altogether?

Reply:
yup agreed on the last point